include/directx/d3dx12_pipeline_state_stream.h

```diff
@@ -1347,6 +1347,45 @@ inline HRESULT D3DX12ParsePipelineStream(const D3D12_PIPELINE_STATE_STREAM_DESC&
         return E_INVALIDARG;
     }
 
+    static const UINT SizesOfSubobject[] = {
```

Comment:
As written, this will end up in the data segment of every .obj file which is a bit of duplication.

@jenatali Would you prefer we use something like ``extern const __declspec(selectany) UINT g_SizeOfSubobject[]`` here -or- create a .cpp file that hosts the one copy of it?

Reply:
I don't like either of those approaches. I'm honestly not a huge fan of using a data table here that's separate from the switch. I don't see the point of it.

If ASAN says there's a bug let's just fix the bug, I assume it's a copy/paste bug, we don't need a paradigm shift.